x/stakeibc/types/params.go

```diff
@@ -19,11 +19,11 @@ var (
 	DefaultStrideCommission              uint64 = 10
 	DefaultValidatorRebalancingThreshold uint64 = 100 // divide by 10,000, so 100 = 1%
 	// 10 minutes
-	DefaultICATimeoutNanos  uint64 = 600000000000
-	DefaultBufferSize       uint64 = 5   // 1/5=20% of the epoch
-	DefaultIbcTimeoutBlocks uint64 = 300 // 300 blocks ~= 30 minutes
-	DefaultFeeTransferTimeoutNanos  uint64 = 600000000000 // 10 minutes
-
+	DefaultICATimeoutNanos         uint64 = 600000000000
+	DefaultBufferSize              uint64 = 5            // 1/5=20% of the epoch
+	DefaultIbcTimeoutBlocks        uint64 = 300          // 300 blocks ~= 30 minutes
+	DefaultFeeTransferTimeoutNanos uint64 = 600000000000 // 10 minutes
+	DefaultMaxICACallsPerEpoch     uint64 = 100
```

Comment:
```suggestion
	DefaultMaxICAMsgDelegateCallsPerEpoch     uint64 = 100
```
this only limits the staking calls afaict

Reply:
renamed, ty!

Reply:
renamed, ty!